classes/class-wc-connect-jetpack.php

```diff
@@ -22,11 +22,11 @@ public static function get_blog_access_token() {
 		}
 
 		/**
-		 * Helper method to get if Jetpack is in development mode
+		 * Helper method to get if Jetpack is in offline mode
 		 *
 		 * @return bool
 		 */
-		public static function is_development_mode() {
+		public static function is_offline_mode() {
```

Comment:
Renamed this method to be in line with the underlying Jetpack Connection method.